internal/hierarchyconfig/validator.go

```diff
@@ -129,6 +132,38 @@ func (v *Validator) handle(ctx context.Context, log logr.Logger, req *request) a
 		return webhooks.Deny(metav1.StatusReasonForbidden, reason)
 	}
 
+	allErrs := field.ErrorList{}
```

Comment:
This function's getting a bit long, maybe move this all to either a `validateMangedMeta` or two `validateManagedLabels`/`validateManagedAnnotations` functions?